attributes/default.rb

```diff
@@ -3,17 +3,20 @@
 
 case node['platform']
 when 'debian', 'ubuntu'
-  default['pgpool']['config']['package_name'] = 'pgpool2'
+  default['pgpool']['config']['package_name'] = 'pgpool-II-pg93' # Might override
```

Comment:
I'm presuming the note below applies here as well, to ensure a 9.3 pgpool is pulled?